### ISSUE TITLE
[Accessibility] Tooltips under user settings for extensions missing aria attributes.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227263]
**Issue**: The "Applies to all profiles" button which opens the tooltip is missing role information and aria-describedby attribute.

**Impact**: Screen reader users will be unable to determine that these tooltips are present.

[Code Reference]
`<span class="setting-item-overrides setting-indicator" tabindex="0" style="display: inline;">Applies to all profiles</span>`

**Steps to repro:**
1. Open VScode Editor
2. Go to User settings -> For any extension setting, check for a tooltip like "Applies to all Profiles"
3. The Accessibility screen reader doesn't announce the presence of a **ToolTip** inside the Toolbar.

For tooltips, the following information is expected:
- The control that opens the tooltip must have aria-describedby set to the ID of the tooltip element.
- The tooltip element must have role="tooltip".

I believe the Tooltip is being rendered by VSCode due to the `Scope` property being added in `contributes.configuration`
Reference: https://code.visualstudio.com/api/references/contribution-points#contributes.configuration

**PFA of the issue:**
![Screenshot 2024-08-30 at 5 16 20 PM](https://github.com/user-attachments/assets/89b94d36-09ee-4eaf-9053-890edc4def52)
